src/resources/benefitCompany.ts

```diff
@@ -6,9 +6,9 @@ import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module
 export const BenefitCompanyResource: ResourceIF = {
   entityType: CorpTypeCd.BENEFIT_COMPANY,
   displayName: GetCorpFullDescription(CorpTypeCd.BENEFIT_COMPANY),
-  title: 'Benefit Company Statement:',
```

Comment:
This caused a double colon.

Reply:
![image](https://user-images.githubusercontent.com/10002889/128761217-d62595eb-6b0e-4f0c-b33e-b041c6b03d29.png)

Reply:
![image](https://user-images.githubusercontent.com/10002889/128761298-d9d067b2-b747-49f8-bb0b-5a7cb70a51c0.png)